frontend2/src/components/tables/chart/TeamChart.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import Highcharts from "highcharts";
+import HighchartsReact from "highcharts-react-official";
+import HighchartsAccessibility from "highcharts/modules/accessibility";
+import * as chart from "./chart";
+import * as random_data from "./random_data";
+
+type UTCMilliTimestamp = number;
+
+export interface TeamChartProps {
+  yAxisLabel: string;
+
+  values: Record<string, Array<[UTCMilliTimestamp, number]>>;
```

Comment:
coudl be helpful to add a comment here explaining what the structure of values is (e.g. maps team names to a list of timestamps and ranking at that timestamp)